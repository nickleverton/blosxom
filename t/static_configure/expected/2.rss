<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
  <channel>
    <title>My Weblog</title>
    <link>http://localhost/2.rss</link>
    <description>Yet another Blosxom weblog.</description>
    <language>en</language>
    <docs>http://blogs.law.harvard.edu/tech/rss</docs>
    <generator>blosxom/2.1.2+dev</generator>

  <item>
    <title>Lorem ipsum II</title>
    <pubDate>Sat, 30 Sep 2006 12:35:00 +0000</pubDate>
    <link>http://localhost/2006/09/30#2</link>
    <category></category>
    <guid isPermaLink="false">http://localhost/2</guid>
    <description>
Lorem ipsum dolor sit amet ipso facto.Lorem ipsum dolor sit amet ipso
facto.Lorem ipsum dolor sit amet ipso facto. Lorem ipsum dolor sit
amet ipso facto.Lorem ipsum dolor sit amet ipso facto. 

Lorem ipso hoccer foer tyr wuz de ras cora dola tymon ipso hoc cer fuz
de ramas cora dola tymon ipso hoccer. Lorem ipso foer tyr wuz de ramas
cora dola tym ipso hor.Lorem ipso foer tyr wuz de ramas cora dola tym
ipso hor tyr wuz de ramas cora dola tymon ipso foer tyr wuz de ramas
cora dola tymo. </description>
  </item>
  </channel>
</rss>
