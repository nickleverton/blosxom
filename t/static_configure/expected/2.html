<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
          "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html>
    <head>
        <link rel="alternate" type="application/rss+xml" title="RSS" href="http://localhost/index.rss" />
        <title>My Weblog</title>
    </head>
    <body>
        <h1>My Weblog</h1>
        <h2>Yet another Blosxom weblog.</h2>
        <h3>2006-09-30</h3>
        <h4><a name="2">Lorem ipsum II</a></h4>
        <div>
Lorem ipsum dolor sit amet ipso facto.Lorem ipsum dolor sit amet ipso
facto.Lorem ipsum dolor sit amet ipso facto. Lorem ipsum dolor sit
amet ipso facto.Lorem ipsum dolor sit amet ipso facto. 

Lorem ipso hoccer foer tyr wuz de ras cora dola tymon ipso hoc cer fuz
de ramas cora dola tymon ipso hoccer. Lorem ipso foer tyr wuz de ramas
cora dola tym ipso hor.Lorem ipso foer tyr wuz de ramas cora dola tym
ipso hor tyr wuz de ramas cora dola tymon ipso foer tyr wuz de ramas
cora dola tymo. </div>
        <p>posted at: 12:35 | path: <a href="http://localhost"> </a> | <a href="http://localhost/2006/09/30#2">permanent link to this entry</a></p>

    </body>
</html>
